helm/chaos-mesh/values.yaml

```diff
@@ -21,7 +21,7 @@ controllerManager:
   replicaCount: 1
 
   image: pingcap/chaos-mesh:latest
-  imagePullPolicy: Always
+  imagePullPolicy: Never
```

Comment:
why change to Never

Reply:
It was temporarily modified during the test and I forgot to change it back.